check/configuration/supported/actionlint.md

```diff
@@ -0,0 +1,22 @@
+---
+description: Actionlint- A Linter for Github Actions
+---
+
+# Actionlint
+
+Actionlint is a linter for infra.
```

Comment:
Imo we should be more specific here (e.g. actionlint is an infra linter for GitHub Actions)